stretch-assignment/digital_timer/index.js

```diff
@@ -0,0 +1,140 @@
+//Variable Declarations
+const html = document.querySelector("html");
+
+const body = document.querySelector("body");
+
+const digits = Array.from(document.querySelector(".digits").children).filter(
+    (elem, i) => i !== 2
+  ),
+  colon = document.getElementById("colon"),
+  startVal = 0;
+
+const btnContainer = document.createElement("div"),
+  start = document.createElement("button"),
+  reset = document.createElement("button"),
+  mainDiv = document.querySelector("div");
```

Comment:
This will grab the first one, naturally, but I would recommend putting a class or ID on this div so you can get it a little more explicitly just in case someone ever screws with your code